src/fragments/lib/auth/flutter/getting_started/40_authenticator.mdx

```diff
@@ -0,0 +1,53 @@
+## Authentication with Amplify
+
+There are 2 options to add authentication capabilities to your application:
+- Option 1: Use Amplify Authenticator UI Library (Developer Preview)
+- Option 2: Call Authentication APIs manually
+
+## Option 1: Use Authenticator UI Library
+
+<Callout>
+
+Amplify Authenticator for `amplify-flutter` is in developer preview.
+
+</Callout>
+
+The Amplify Authenticator UI library allows you to quickly create an authentication/registration flow within your app. The experience generated within your app will be determined by the `amplifyconfiguration.dart` file that is automatically generated for you after initializing your app, and adding auth or other categories. Here is an [example app](https://github.com/aws-amplify/amplify-flutter/tree/main/packages/amplify_authenticator/example) that utilizes Authenticator.
+
+First, you need to add `amplify-authenticator` as a dependency in your `pubspec.yaml` file in your root folder of your app. 
+```yaml
+dependencies:
+  amplify_flutter: ^0.3.0
```

Comment:
@dnys1 Already noted this - these dependencies need to reference the rc release.

Reply:
```yaml
amplify_flutter: ^0.3.0-rc.3
amplify_auth_cognito: ^0.3.0-rc.3
amplify_authenticator: ^0.1.0-rc.1
```

Reply:
changed now